modules/qualify/qualify.c

```diff
@@ -0,0 +1,300 @@
+/**
+ * @file qualify.c Pinging of peer in CALL_STATE_INCOMING via SIP OPTIONS
+ *
+ * Copyright (C) 2023 Commend.com - m.fridrich@commend.com
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <re.h>
+#include <baresip.h>
+
+
+/**
+ * @defgroup qualify module
+ *
+ * This module implements a pinging mechanism using SIP OPTIONS to qualify the
+ * peer while a call is in the INCOMING state to ensure that the peer is
+ * reachable and we will be able to successfully answer the call.
+ *
+ * Configuration:
```

Comment:
Instead of "Configuration:" write "Configure in address parameter `extra`:"

Reply:
Done.